google/cloud/aiplatform/metadata/experiment_run_resource.py

```diff
@@ -990,6 +993,103 @@ def log_metrics(self, metrics: Dict[str, Union[float, int, str]]):
             # TODO: query the latest metrics artifact resource before logging.
             self._metadata_node.update(metadata={constants._METRIC_KEY: metrics})
 
+    def log_classification_metrics(
+        self,
+        *,
+        labels: Optional[List[str]] = None,
+        matrix: Optional[List[List[int]]] = None,
```

Comment:
Won't change because it is consistent with [KFP SDK](https://github.com/kubeflow/pipelines/blob/166d6bb917cff810c4b7fdf6591ec1affb6ae5f7/sdk/python/kfp/components/types/artifact_types.py#L309)